code/datums/martial/krav_maga.dm

```diff
@@ -86,7 +86,7 @@
 	D.visible_message("<span class='warning'>[A] pounds [D] on the chest!</span>", \
 				  	"<span class='userdanger'>[A] slams your chest! You can't breathe!</span>")
 	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
-	D.losebreath += 5
+	D.losebreath = Clamp(D.losebreath + 5, 0, 10)
```

Comment:
Something like
```
if D.losebreath < 10
    D.losebreath = Clamp(D.losebreath + 5, 0, 10) //effectively min(D.losebreath + 5, 10)
```
so that the punch doesn't add losebreath if the losebreath is already higher than 10, which is what I assume you want.